examples/psyir/README.md

```diff
@@ -63,3 +64,14 @@ To run this example:
 ```sh
 > python create_aggregate_types.py
 ```
+
+## Example 3:
+
+Demonstrates how to manipulate an existing PSyIR tree. This example
+imports the PSyIR created in Example 1, applies some modifications
+to it and then generated the new modified output code. This example may
```

Comment:
and then outputs the modified PSyIR as Fortran code.

Reply:
Done